CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 14.0.0
+
+#### Breaking changes - [#525](https://github.com/openfisca/openfisca-core/pull/525)
+
+* Rename Variable attribute `url` to `reference`
+  - As previously, this attribute can be a string, or a list of strings.
+  - Variable attribute `law_reference` is removed.
+  - `TaxBenefitSystem.reference` renamed to `TaxBenefitSystem.reference_tbs`.
```

Comment:
Not sure how I feel about that. I understand how you came to do this renaming (to be consistent about what a `reference` means), but this is not directly related to our issue.


@MattiSG thoughts ?